osbs/cli/main.py

```diff
@@ -416,6 +461,32 @@ def _display_build_summary(build):
     return return_val
 
 
+def _display_pipeline_run_summary(pipeline_run):
+    output = [
+        "",  # Empty line for cleaner display
+        "pipeline run {} is {}".format(pipeline_run.pipeline_run_name, pipeline_run.status_reason),
+    ]
+    return_val = -1
+
+    annotations = pipeline_run.get_info()['metadata']['annotations']
+
+    if pipeline_run.is_succeeded():
+        return_val = 0
+        all_repositories = annotations.get('repositories', {})
+
+        for kind, repositories in all_repositories.items():
+            if not repositories:
+                continue
+            output.append('{} repositories:'.format(kind))
+            for repository in repositories:
+                output.append('\t{}'.format(repository))
+
+    for line in output:
+        print(line)
+
+    return return_val
```

Comment:
Does this function need to return "return_val" as number? is this expected by the consumer of the method or could this be changed?

Reply:
now it doesn't return anything just prints

Reply:
It's simply returned from `_print_pipeline_run_logs()`: `return _display_pipeline_run_summary(pipeline_run)` - I agree with Max that

```python
retval = _display_pipeline_run_summary(pipeline_run)
return retval
```

(in `_print_pipeline_run_logs()`) is better

Reply:
I've changed it !

Reply:
I feel that not calling _display_pipeline_run_summary at all in _print_pipeline_run_logs would be better and improve readability as they sort of do differentthing. and I would not return anything from any of those methods, so the result would be:
```
def cmd_build_source_container(args):
    ....
    _print_pipeline_run_logs(args, pipeline_run)
   _display_pipeline_run_summary(pipeline_run)
  return pipeline_run.has_suceeded() # assuming we are ok with returning this, and we don't need that special numbers 0/-1



```